lib/we/call/connection.rb

```diff
@@ -49,20 +54,25 @@ def new(host:, timeout: nil, open_timeout: OPEN_TIMEOUT, app: guess_app, env: gu
       def create
         builder = QueryableBuilder.new(&Proc.new { |_| })
 
-        Faraday.new(url: host, builder: builder) do |faraday|
-          faraday.headers['User-Agent'] = app
-          faraday.headers[config.app_name_header] = app
-          faraday.headers[config.app_env_header] = env
-          faraday.options[:timeout] = timeout
-          faraday.options[:open_timeout] = open_timeout
+        headers = {
```

Comment:
Messing with the options property directly was a little scary as it changed over time. This approach is 100% gonna work over more versions.